catalog/core/catalog-core-backupplugin/src/main/java/ddf/catalog/backup/CatalogBackupPlugin.java

```diff
@@ -69,45 +75,59 @@ public CatalogBackupPlugin() {
      * Backs up created metacards to the file system backup.
      *
      * @param input the {@link CreateResponse} to process
-     * @return
+     * @return {@link CreateResponse}
      * @throws PluginExecutionException
      */
     @Override
     public CreateResponse process(CreateResponse input) throws PluginExecutionException {
+
         if (enableBackupPlugin && Requests.isLocal(input.getRequest())) {
-            LOGGER.debug("Performing backup of metacards in CreateResponse.");
+            LOGGER.trace("Performing backup of metacards in CreateResponse.");
 
             if (rootBackupDir == null) {
                 throw new PluginExecutionException("No root backup directory configured.");
             }
 
-            List<String> errors = new ArrayList<String>();
+            EXECUTOR_SERVICE.submit(() -> processBatch(input.getCreatedMetacards()));
 
-            List<Metacard> metacards = input.getCreatedMetacards();
+            //Stop accepting new tasks
+            EXECUTOR_SERVICE.shutdown();
```

Comment:
I don't think you want to shut it down here. If you shut it down it won't accept anything again.